### ISSUE TITLE
[Issue]: 10.8.8 Clean Install - IMDB ratings missing / not correct

### ISSUE THREAD

**Issue by Oatavandi** [jellyfin/jellyfin#7933]
### Please describe your bug

Jellyfin Version: 10.8.8
Synology NAS - Docker

Two Issues, I faced while doing a clean Install of 10.8.0

IMDB ratings are not fetched or ratings are incorrect with 10.8.8. Did a clean install with 10.7.7 and IMDB ratings are correct.

### Jellyfin Version

10.8.8

### Environment

```markdown
- OS:Synology NAS - Docker
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
Please provide logs.

**Comment by Oatavandi** [comment]
10.8.0 IMDB Ratings - Screenshot Attached. (For 10.7.7 - the ratings are perfect)

![10 8 0_IMDB_Ratings_Screenshot](https://user-images.githubusercontent.com/70758372/173850023-502094af-c4e2-43c3-91bf-18a1ca9097f3.jpg)

**Comment by Oatavandi** [comment]
2. Logs will attach soon - scanning in progress (doing a fresh install again for 10.8.0)

**Comment by mastercho** [comment]
additional to IMDB i want to add that if i set different language to fetch than English, it's not fetching English like in 10.7.7 if my proffered language is not exist

**Comment by Oatavandi** [comment]
The issue is still not fixed with 10.8.8

IMDB ratings are not fetched or ratings are incorrect with 10.8.8. Did a clean install with 10.7.7 and IMDB ratings are correct.
